packages/react-dev-utils/launchEditor.js

```diff
@@ -144,8 +158,10 @@ function guessEditor() {
     return shellQuote.parse(process.env.REACT_EDITOR);
   }
 
-  // Using `ps x` on OSX or `Get-Process` on Windows we can find out which editor is currently running.
-  // Potentially we could use similar technique for Linux
+  // We can find out which editor is currently running by:
+  // `ps x` on OSX
+  // `Get-Process` on Windows
+  // `ps -e` on Linux
```

Comment:
I assume this should be updated to say `ps x`? Or update above comment to say:
> `ps x` on macOS and Linux

Reply:
changed it